packages/itwin/tree-widget/src/tree-widget-react/components/trees/classifications-tree/ClassificationsTree.tsx

```diff
@@ -31,13 +36,34 @@ export function ClassificationsTree({
   emptyTreeContent,
   getDecorations,
   getActions,
+  onLabelChanged,
 }: ClassificationsTreeProps) {
   const { categoriesTreeProps, rendererProps } = useClassificationsTree({
     activeView,
     hierarchyConfig,
     emptyTreeContent,
   });
 
+  const nodeActions = useCallback(
+    (node: PresentationHierarchyNode) => {
+      return [<RenameAction key="RenameAction" />, ...(getActions ? getActions(node) : [])];
```

Comment:
I don't think we should be adding this action ourselves - we'll have consumers that show this tree as read-only, so they won't want it.

Instead, I think we should just add a piece of docs next to `onLabelChanged` / `getEditingProps`, explaining that label editing requires adding a `<RenameAction />`.

Reply:
I agree that docs in `@itwin/presentation-hierarchies-react` is probably the best place to explain this, since both the `<RenameAction />` and `getEditingProps` prop are defined there.